data/releasenotes.json

```
@@ -1,4 +1,17 @@
 [
+	{
+		"version": "alpha/2024-11-21",
+		"date": "2024-11-21",
+		"breaksdb": false,
+		"notes": [
+			"<b>New Feature:</b> Add support for <tt>psql</tt> builtin describe commands <tt>\\d table</tt> and <tt>\\d+ table</tt>.",
+			"<b>PostgreSQL compatibility:</b> Include attribute defaults in the <tt>pg_attrdef</tt> system table.",
+			"<b>Maintenance:</b> Fixed a rare deadlock during concurrent compression of inserts.",
+			"<b>Maintenance:</b> Enforce an upper limit for the maximum number of WAL files. When the overall log size becomes too large, writing transactions are now throttled.",
```

```suggestion
			"<b>Performance:</b> Reduce transaction tail latency: When the overall write-ahead log size becomes too large, writing transactions are now throttled instead of paused.",
```
Wir können ja immer noch kein limit enforcen. Dafür ist das ja eigentlich schon ein performance-win?